src/app/backend/allobjects/allobjects.go

```diff
@@ -0,0 +1,100 @@
+	"github.com/kubernetes/dashboard/src/app/backend/resource/secret"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/service"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/statefulset"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/workload"
+	"k8s.io/client-go/kubernetes"
+)
+
+// ObjectResult is a list of objects present in the given namespace
+type ObjectResult struct {
+	// Config and storage.
+	ConfigMapList             configmap.ConfigMapList       `json:"configMapList"`
+	PersistentVolumeClaimList pvc.PersistentVolumeClaimList `json:"persistentVolumeClaimList"`
+	SecretList                secret.SecretList             `json:"secretList"`
+
+	// Discovery and load balancing.
+	ServiceList service.ServiceList `json:"serviceList"`
+	IngressList ingress.IngressList `json:"ingressList"`
+
+	// Workloads.
+	DeploymentList            deployment.DeploymentList    `json:"deploymentList"`
+	ReplicaSetList            replicaset.ReplicaSetList    `json:"replicaSetList"`
+	JobList                   job.JobList                  `json:"jobList"`
+	ReplicationControllerList rc.ReplicationControllerList `json:"replicationControllerList"`
+	PodList                   pod.PodList                  `json:"podList"`
+	DaemonSetList             daemonset.DaemonSetList      `json:"daemonSetList"`
+	StatefulSetList           statefulset.StatefulSetList  `json:"statefulSetList"`
+
+	// TODO(maciaszczykm): Third party resources.
+}
+
+func GetAllObjects(client *kubernetes.Clientset, metricClient metricapi.MetricClient,
+	nsQuery *common.NamespaceQuery,
+	dsQuery *dataselect.DataSelectQuery) (*ObjectResult, error) {
+
+	configResources, err := config.GetConfig(client, nsQuery, dsQuery)
+	if err != nil {
+		return &ObjectResult{}, err
+	}
+
+	discoveryResources, err := discovery.GetDiscovery(client, nsQuery, dsQuery)
+	if err != nil {
+		return &ObjectResult{}, err
+	}
+
+	workloadsResources, err := workload.GetWorkloads(client, metricClient, nsQuery, dsQuery)
+	if err != nil {
+		return &ObjectResult{}, err
+	}
+
+	return &ObjectResult{
+		// Config and storage.
+		ConfigMapList:             configResources.ConfigMapList,
+		PersistentVolumeClaimList: configResources.PersistentVolumeClaimList,
+		SecretList:                configResources.SecretList,
+
+		// Discovery and load balancing.
+		ServiceList: discoveryResources.ServiceList,
+		IngressList: discoveryResources.IngressList,
+
+		// Workloads.
+		DeploymentList:            workloadsResources.DeploymentList,
+		ReplicaSetList:            workloadsResources.ReplicaSetList,
+		JobList:                   workloadsResources.JobList,
+		ReplicationControllerList: workloadsResources.ReplicationControllerList,
+		PodList:                   workloadsResources.PodList,
+		DaemonSetList:             workloadsResources.DaemonSetList,
+		StatefulSetList:           workloadsResources.StatefulSetList,
+	}, nil
+}
```

Comment:
@cnwalker Hmm, okay. I didn't notice, that cluster resources are missing. Still, you can try to export it somehow to avoid code duplication :)